.travis.yml

```diff
@@ -4,39 +4,39 @@ cache:
   directories:
   - $HOME/gnatsd
   - $HOME/nats-streaming-server
+  - $HOME/pbuf
 
-sudo: false
+sudo: true
 
-compiler: 
+compiler:
   - gcc
   - clang
 
-addons:
-  apt:
-    packages:
-    - valgrind
-    - libssl-dev
-    - unzip
-
 before_install:
+  - bash install_deps.sh
   - eval "${MATRIX_EVAL}"
-  - bash install_servers.sh
 
 before_script:
-  - export PATH=$HOME/gnatsd:$HOME/nats-streaming-server:$PATH
+  - export PATH=$HOME/gnatsd:$HOME/nats-streaming-server:$HOME/pbuf:$PATH
```

Comment:
Lets migrate gnatsd -> nats-server

Reply:
I could not upgrade past 1.3.0 due to valgrind on this system that is failing for SSL tests. This is a problem with valgrind and would need to install valgrind 1.3.12+ which is unavail on this system. However, since I have removed valgrind for the use of `-fsanitize=address`, I should be able to upgrade to latest nats-server. Will do so now.